app/assets/javascripts/utils/model_utils.js

```diff
@@ -40,7 +40,14 @@ export const sortByKey = (models, sortKey, previousKey = null, desc = false) =>
 
   const reverse = !sameKey !== !desc; // sameKey OR desc is truthy, but not both
   let newModels;
-  if (reverse) {
+  if (absolute) {
+    const sorted = _.sortBy(models, [o => Math.abs(o[sortKey])]);
+    if (reverse) {
```

Comment:
This logic could probably be refactored to handled reverse separately from the rest. Like, instead of `if/else if/else`, you first do the basic sorting with whatever options you need, and then you reverse the results at the end if that option is enabled.

Reply:
There's a problem with that. We can't simply reverse the sorted because the `sort` function is specifically written to take care of `null` values to end at last instead of beginning when descending order is to be obtained.